openmc/filter.py

```diff
@@ -434,20 +434,19 @@ def get_pandas_dataframe(self, data_size, **kwargs):
         return df
 
 
-class IntegralFilter(Filter):
-    """Tally modifier that describes phase-space and other characteristics.
+class UniverseFilter(Filter):
+    """Bins tally event locations based on the Universe they occured in.
 
     Parameters
     ----------
-    bins : Integral or Iterable of Integral
-        The bins for the filter. This takes on different meaning for different
-        filters. See the docstrings for sublcasses of this filter or the online
-        documentation for more details.
+    bins : openmc.Universe, Integral, or iterable thereof
+        The Universes to tally.  Either openmc.Universe objects or their
```

Comment:
Equal-spaced font, have to hit the carriage return at the end of the line... might as well be :p 

I was curious about this and found there is [a whole Wiki article](https://en.wikipedia.org/wiki/Sentence_spacing) on the topic.  It has this amusing statement in the intro, "The debate continues on the World Wide Web."